tests/dragonfly/replication_test.py

```diff
@@ -1685,7 +1685,7 @@ async def test_network_disconnect_small_buffer(df_local_factory, df_seeder_facto
             # Df is blazingly fast, so by the time we tick a second time on
             # line 1674, DF already replicated all the data so the assertion
             # at the end of the test will always fail
-            fill_task = asyncio.create_task(seeder.run())
+            fill_task = asyncio.create_task(seeder.run(target_ops=40000000))
```

Comment:
why did you add target_ops?
The flow of the test should be seeder run until we call below seeder.stop()

What was failing that caused you to change this?

Reply:
the test failed on the build machine, but locally everything was correct. So according to comments I've increased target_ops

Reply:
This comment was written in this PR https://github.com/dragonflydb/dragonfly/pull/1970/files and I believe its not relevant now as we dont use target ops. We need to find why this failed but for now please remove this change.